Add Plantas nuevas y mutaciones

Spircen opened the pull request:
## What Does This PR Do
Agrega múltiples nuevas plantas y mutaciones a petición de Hessica quien ya contaba con todos sus sprites hechos y las ideas de que deberían de hacer cada una de estas. cambia la dirección de sprites del money tree y ordena alfabéticamente las semillas en la vendedora. Las nuevas plantas son: 

- Agave: Sirve para destilar Tequila con el barril de fermentado
- Anona: Tiene mutación a Reticulata y produce jugo de Anona
- Anonna Reticulata: Produce mutageno inestable 
- Prickly Pear: Cactus dulce con bastantes nutrientes, puede mutar a San Pedro
- San Pedro: Cactus bastante nutritivo que puede hacer bastante daño al ser lanzado, muta a Peyote
- Peyote: Cactus alucinógeno puramente LSD 
- Bombonaut: Mutación del coco, pero explosivo
- Kiwi: Fruta dulce que puede generar jugo de Kiwi y muta a un huevo de kiwi
- Actual Kiwi: Fruta que al ser usada genera un Kiwi
- Mango: Fruta dulce que genera jugo de Mango
- Mate: Planta que permite generar futuras bebidas
- Champignon: Mutación de Chanterelle, hongo sin toxinas para pizzas.
- Nispero: Fruta perfecta para bebidas alcoholicas
- Peach: Linda fruta bastante THICC
- Bell Pepper: Perfecta para tus pizzas
- Ricinus: Permite crear Castor Oil reagent que cura daño bruto y quemaduras.
- Strawberry : La deliciosa fruta de la pasión. 
- Avocado: Perfecta para emparedados y sus mutaciones

Aun debo de modificar las etapas de crecimiento de cada una y modificar algunos sprites que ya no ocupare pero las plantas son funcionales sin problemas. 

A la par cambia los sprite de Garlic. 

## Why It's Good For The Game
Agrega nueva variedad a la multitud de plantas que ya existen a la par que genera plantas que son bastantes conocidas en la vida real pero inexistentes, a la par que varias personas suelen tener ideas perfectas que generan para sus personajes pero no se logran cumplir por falta de algunas plantas de este catalogo y... casi no suelo hacer PRs de comida, variedad, diversión y mas sabor. 

## Images of changes

                                    Nuevas Plantas
![image](https://user-images.githubusercontent.com/46639834/77839267-64b65680-7138-11ea-8472-8b0e84ec1fc3.png)

                                 Agregamos a maquina
![image](https://user-images.githubusercontent.com/46639834/77839282-80216180-7138-11ea-8050-213c0dbf3d28.png)

                              Avocado Stuff and Actual Kiwi
![image](https://user-images.githubusercontent.com/46639834/79054585-caf8aa00-7c0b-11ea-8ca7-cc623789fbf3.png)

![image](https://user-images.githubusercontent.com/46639834/79054596-db108980-7c0b-11ea-8131-c9d264cf38da.png)

                                         Nuevos Jugos (Menos HotLine Bling)
![image](https://user-images.githubusercontent.com/46639834/79083649-2eaad200-7cf5-11ea-8db6-d2a0247e9266.png)

## Changelog
:cl:
add: Nuevas plantas y mutaciones
add: Nuevos Jugos
tweak: Garlic Sprite
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Spircen committed:
Avance

Spircen committed:
Merge branch 'master' into hes-stuff

Spircen committed:
Part 2 Framework Hess Stuff

Spircen committed:
Merge branch 'master' into hes-stuff

Spircen committed:
Agave Completado

Spircen committed:
Anonna Completado

Spircen committed:
Cactus Completado y Menores Cambios a Anonna

Spircen committed:
Bombonaut explosion fix

Spircen committed:
Kiwi Completado

Spircen committed:
Mango Completado

Spircen committed:
Mate Completado

Spircen committed:
Champignon Completado

Spircen committed:
Nispero Completado

Spircen committed:
Ultimas plantas completadas y cambios de desc para semillas

Spircen committed:
avoid delete

Spircen committed:
Merge remote-tracking branch 'upstream/master' into hes-stuff

Spircen committed:
Fixes y Add a Seed Vendor

DanaDririon commented on code/HISPANIA/modules/hydroponics/grown/cactus.dm:
```diff
@@ -0,0 +1,72 @@
+	icon_dead = "peyote_dead"
+	product = /obj/item/reagent_containers/food/snacks/grown/prickly_pear/san_pedro/peyote
+	mutatelist = list(/obj/item/seeds/prickly_pear/san_pedro/peyote)
+	reagents_add = list("lsd" = 1)
```
Ojo, si le pones 1 significa 100%, lo que significa a su vez que tendrá las mismas unidades del reagente que la potencia.
80 potencia = 80u de lsd

Spircen replied:
Si, es Peyote. Se supone que siempre tenga esa cantidad máxima de drogas.

DanaDririon replied:
top kek

DanaDririon commented:
No agranda la química de hydro, pero me gustan las plantas
+1

Spircen committed:
Merge remote-tracking branch 'upstream/master' into hes-stuff

Spircen committed:
Last Fixes

-Borramos los sprites que no se ocupan
-Cambiamos variables para cada planta
-Formato correcto de sprites
-Reposicionamos sprites
-Colores a Reagents

Spircen committed:
Galalalala

-Borramos lineas inecesarias por cambio de Sprite de garlic

Spircen committed:
Son demasiadas plantas y se me paso un detalle en kiwis

Kiwis ocupan spray base de kiwi pero semilla y fruta es distinta

SagaStars commented:
Ahora hay que sacarle buen provecho al chef y al bartender

DanaDririon commented on code/HISPANIA/modules/hydroponics/grown/cactus.dm:
```diff
@@ -0,0 +1,67 @@
+	species = "peyote"
+	plantname = "Peyote Cactus"
+	product = /obj/item/reagent_containers/food/snacks/grown/prickly_pear/san_pedro/peyote
+	mutatelist = list(/obj/item/seeds/prickly_pear/san_pedro/peyote)
```
Aquí creo que el mutate list lo debes dejar vacío, porque esto está haciendo que la planta mute en sí misma.

Spircen replied:
Ah cierto, aquí se me paso la mutación.

DanaDririon commented on code/HISPANIA/modules/hydroponics/grown/coconut.dm:
```diff
@@ -1,18 +1,16 @@
 /obj/item/seeds/coconut
 	name = "pack of coconut seeds"
-	desc = "These seeds grow into coconut palms."
+	desc = "These seeds grow into cocount palms"
```
typo

Spircen replied:
Ya lo cambio

DanaDririon commented on code/HISPANIA/modules/hydroponics/grown/kiwi.dm:
```diff
@@ -0,0 +1,67 @@
+	weed_chance = 60
+	growing_icon = 'icons/hispania/obj/hydroponics/growing_fruits.dmi'
+	reagents_add = list("kiwi" = 0.19, "nutriment" = 0.12)
+
```
Aquí también tienes que poner la lista de mutación vacía, porque lo está heredando el del otro.
También puedes borrar las variables species, reagents_add y growing icon, que las hereda

Spircen replied:
Vale, voy a hacer eso con las plantas que mutan.

DanaDririon commented:
El resto se ve bien 👍 
Mientras que pruebes todas las plantas y que funcionen bien, apruebo

Spircen committed:
Fix a mutaciones y typo

DanaDririon commented on code/HISPANIA/modules/hydroponics/grown/coconut.dm:
```diff
@@ -1,18 +1,16 @@
 /obj/item/seeds/coconut
 	name = "pack of coconut seeds"
-	desc = "These seeds grow into coconut palms."
+	desc = "These seeds grow into cocount palms."
```
Me refería a _coco**un**t_ xd

Spircen replied:
Ah ya, ahi quedo xd

DanaDririon commented on code/HISPANIA/modules/hydroponics/grown/peach.dm:
```diff
@@ -0,0 +1,26 @@
+	lifespan = 20
+	endurance = 25
+	yield = 9
+	potency = 9
```
Y también encuentro que en general pones potencia baja
Recuerda que este valor va de 0 a 100, entonces entre potencia 3 o potencia 9 no hay mucha diferencia

Spircen replied:
Ya cambie el resto de todos en potency

Spircen committed:
Cambios de Potency y typo

Spircen committed:
Merge branch 'master' into hes-stuff

Spircen committed:
Add Semillas a Vending

DanaDririon commented on code/HISPANIA/modules/hydroponics/grown/peach.dm:
```diff
@@ -0,0 +1,26 @@
+	lifespan = 20
+	endurance = 25
+	yield = 9
+	potency = 75
```
Potencia 75 sin tener que mutar una planta :dab:

Spircen replied:
New meta chi cheñol

Spircen committed:
test

Spircen committed:
test

Spircen committed:
¿Espacio?

Spircen committed:
Merge branch 'master' into hes-stuff

Spircen committed:
Merge branch 'master' into hes-stuff

Spircen committed:
Merge branch 'master' into hes-stuff

Spircen committed:
testeando

DanaDririon committed:
Branch Plantas

Branch Plantas

DanaDririon committed:
Avocados

DanaDririon committed:
Unos puntos en las desc

DanaDririon committed:
Ups, esto también

Spircen committed:
Merge pull request #5 from DanaDririon/avocadospircen

Agrega Avocado y ordena unas plantas

Spircen committed:
Merge remote-tracking branch 'upstream/master' into hes-stuff

Spircen committed:
Merge branch 'hes-stuff' of https://github.com/Spircen/Paradise into hes-stuff

Spircen committed:
Bajamos Potencias

Spircen committed:
Jugos no reagents

Spircen committed:
Bebidas y Sprites

Spircen committed:
Hahaha puse direcciones mal

Spircen committed:
Not sure how this work

Spircen commented:
Listo, potencias bajadas, sprites agregados a las bebidas para que no sean vasos de "What...?"

Spircen committed:
Caps

Las plantas no llevan mayusculas al ser objetos

DanaDririon commented:
Apruebo como está el PR ahora 👍

Spircen committed:
Merge branch 'master' into hes-stuff

Spircen committed:
Cambios mencionados por Ume
